ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -107,6 +107,25 @@ public File getStorePasswordFile() {
     }
   }
 
+  static class TlsClientAuthentication {
+    @Option(
+        names = "--tls-known-clients-file",
+        description = "Path to a file containing the fingerprints of authorized clients.",
+        arity = "1",
+        required = true)
+    private final File tlsKnownClientsFile = null;
```

Comment:
Why known-clients-file required = true within the group?

Reply:
this is a mutually exclusive group https://picocli.info/apidocs/picocli/CommandLine.ArgGroup.html